string name
string value
---
int16 status
string message
